### PR TITLE
More futures fixes

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_trading/personal_data/orders/order.py [hunk 1]
```diff
@@ -312,7 +312,7 @@ def get_computed_fee(self, forced_value=None):
             currency = computed_fee[enums.FeePropertyColumns.CURRENCY.value]
         else:
             symbol_fees = self.exchange_manager.exchange.get_fees(self.symbol)
-            fees = decimal.Decimal(f"{symbol_fees[self.taker_or_maker]}")
+            fees = decimal.Decimal(f"{symbol_fees[self.taker_or_maker]}") / constants.ONE_HUNDRED
```
Comment (Herklos) [inline octobot_trading/personal_data/orders/order.py:315]: I think we should uniformize at lower level (just after we call ccxt to request fees)

Comment (GuillaumeDSM) [reply octobot_trading/personal_data/orders/order.py:315]: Good idea 👍

File: octobot_trading/personal_data/positions/position.py [hunk 2]
```diff
@@ -326,10 +326,11 @@ def _update_realized_pnl_from_order(self, order):
         Removes order's fees from realized pnl
         :param order: the realized pnl update
         """
-        if self.symbol_contract.is_inverse_contract():
-            realised_pnl_update = -order.get_total_fees(order.currency)
-        else:
-            realised_pnl_update = -order.get_total_fees(order.market)
+        fees_currency = order.currency if self.symbol_contract.is_inverse_contract() else order.market
+        realised_pnl_update = -order.get_total_fees(fees_currency)
```
Comment (Herklos) [inline octobot_trading/personal_data/positions/position.py:330]: Why removing is_inverse_contract check ?

File: octobot_trading/personal_data/positions/position.py [hunk 3]
```diff
@@ -289,7 +290,8 @@ def update_from_order(self, order):
         :return: the updated quantity, True if the order increased position size
         """
         # consider the order filled price as the current reference price for pnl computation
-        self._update_mark_price(order.filled_price)
+        # do not check liquidation as our position might be closed by this order
```
Comment (Herklos) [inline octobot_trading/personal_data/positions/position.py:296]: :+1:

**Review by Herklos — APPROVED** [review]

**Pull request merged**